pkgs/tools/system/auto-cpufreq/default.nix

```diff
@@ -23,11 +27,17 @@ python3Packages.buildPythonPackage rec {
       inherit version;
     })
 
+     # patch to prevent update
+    ./prevent-update.patch
+
     # patch to prevent script copying and to disable install
     ./prevent-install-and-copy.patch
-    # patch to prevent update
-    ./prevent-update.patch
-  ];
+ ];
+
+  postPatch = ''
+    substituteInPlace auto_cpufreq/core.py --replace '/opt/auto-cpufreq/override.pickle' /var/run/override.pickle
+    substituteInPlace scripts/org.auto-cpufreq.pkexec.policy --replace "/opt/auto-cpufreq/venv/bin/auto-cpufreq" $out/bin/auto-cpufreq
+    '';
```

Comment:
```suggestion
  postPatch = ''
    substituteInPlace auto_cpufreq/core.py --replace '/opt/auto-cpufreq/override.pickle' /var/run/override.pickle
    substituteInPlace auto_cpufreq/gui/app.py --replace '/usr/local/share/auto-cpufreq/images/icon.png' $out/share/auto-cpufreq/images/icon.png
    substituteInPlace auto_cpufreq/gui/app.py --replace '/usr/local/share/auto-cpufreq/scripts/style.css' $out/share/auto-cpufreq/scripts/style.css
    substituteInPlace scripts/org.auto-cpufreq.pkexec.policy --replace "/opt/auto-cpufreq/venv/bin/auto-cpufreq" $out/bin/auto-cpufreq
    '';
```

Reply:
You can now use `--replace-fail` in case of unforeseen upstream changes.